packages/samplePaymentMethod/project.json

```diff
@@ -0,0 +1,25 @@
+{
+  "root": "packages/samplePaymentMethod",
```

Comment:
Let's just remove the `samplePaymentMethod`. It only exists because I don't have sample. Now we have `ApplePay` so there's no need for it anymore.